src/studio/src/designer/DataModeling.Tests/Json/JsonSchemaSeresAnalyzerTests.cs

```diff
@@ -30,6 +32,11 @@ public async Task IsValidComplexType_ComplexType_ShouldReturnTrue(string path, s
             _testOutputHelper.WriteLine($"{testCase}");
 
             var schema = await ResourceHelpers.LoadJsonSchemaTestData(path);
+
```

Comment:
[InlineData(@"Model\JsonSchema\ComplexContentExtension.json...
[InlineData(@"Model\JsonSchema\ComplexSchema.json...

These two are red when I run all tests.

Reply:
I know, its in the description that we need to handle IsComplexType and derivatives